themes/bootstrap3/templates/Helpers/star-rating.phtml

```diff
@@ -0,0 +1,31 @@
+<?php
+  $id = $this->escapeHtmlAttr(uniqid());
+  $labelAttrs = !empty($this->labelAttrs) ? (' ' . implode(' ', $this->labelAttrs)) : '';
+  $inputAttrs = !empty($this->inputAttrs) ? (' ' . implode(' ', $this->inputAttrs)) : '';
+  $rating = $ratingData['count'] ? ($ratingData['rating'] ?? 0) : null;
+  // Unique name for the fields of read-only ratings so that displaying multiple ratings on a page works:
+  $fieldName = $this->readonly ? "rating_{$id}" : 'rating';
+?>
+<div class="star-rating<?=$this->readonly ? ' readonly' : ''?>">
+  <label class="rating__label hidden" for="rating_<?=$id?>_none" aria-label="<?=$this->transEsc('rating_none')?>"<?=$labelAttrs?>>
+    &nbsp;
+  </label>
+  <input class="rating__input hidden" name="<?=$fieldName?>" id="rating_<?=$id?>_none" value="" type="radio" autocomplete="off"<?=null === $rating ? 'checked' :''?><?=$inputAttrs?>>
+  <label class="rating__label" for="rating_<?=$id?>_0" aria-label="<?=$this->transEsc('rating_0')?>"<?=$labelAttrs?>>
+    &nbsp;
```

Comment:
Without any hover highlighting or any other visuals besides a pointer cursor, I find "click the empty space just to the left of the first star" to be a cryptic way to set 0 stars.

Reply:
There's the subtle hover effect of no highlighted star at all, but I agree that it's not too clear. In the light of this, I decided to remove it altogether. I'd say that half-star as a minimum is more clear and obviously "something". What do you think?